packages/components/src/Hero/styledComponents.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import styled, { StyledComponentClass } from "styled-components";
+import { colors, fonts } from "../styleConstants";
+
+export const HeroLabel = styled.span`
+  color: ${colors.basic.WHITE};
+  display: block;
+  font-family: ${fonts.SANS_SERIF};
+  font-size: 20px;
+  font-weight: 800;
+  letter-spacing: -0.27px;
+  margin-bottom: 30px;
+`;
+
+export const HeroHeading = styled.h2`
+  color: ${colors.basic.WHITE};
```

Comment:
Should we move the `color` and `font-family` definitions up to a parent element and save a few bytes? Maybe `HeroOuter`?

The build for the dapp is getting big and saving a couple bytes here and there might be nice